lighty-modules/lighty-restconf-nb-community/src/main/java/io/lighty/modules/northbound/restconf/community/impl/CommunityRestConf.java

```diff
@@ -108,6 +126,19 @@ protected boolean initProcedure() {
                     new ServletContextHandler(contexts, this.restconfServletContextPath, true, false);
             mainHandler.addServlet(jaxrs, "/*");
 
+            final ServletContextHandler notifHandler =
```

Comment:
You can see at https://github.com/opendaylight/netconf/blob/0e30eb81083ea26edd80ed9bece151f903ccfa7b/restconf/restconf-nb/src/main/java/org/opendaylight/restconf/nb/rfc8040/JaxRsNorthbound.java#L101-L102 that `notif` should be in fact `rests/notif`.

Reply:
resolved